From a58bf61c99fe5ba744cc91335838a358356f28a0 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Mon, 10 Jan 2022 16:40:21 +0200
Subject: [PATCH] [SW-0] greco: move uboot binary to unused space

Preboot uses the fist 10MB of SRAM for its memory management. This
colides with uboot being copied to the first 1MB whil loading FIT.
Solved by pushing uboot +16BM (17MB in total)

Change-Id: I1a324fe480657bd0926957303139a5c429aed3e8
Signed-off-by: Guy Nisan <gnisan@habana.ai>
---

diff --git a/board/habanalabs/greco/post-image-fit.sh b/board/habanalabs/greco/post-image-fit.sh
index a515ba3..940292a 100755
--- a/board/habanalabs/greco/post-image-fit.sh
+++ b/board/habanalabs/greco/post-image-fit.sh
@@ -9,7 +9,7 @@
 KERNEL_LOAD_ADDR_LO=0x08080000
 
 # load addresses in ITS file
-UBOOT_ADDR_LO=0xF0100000
+UBOOT_ADDR_LO=0xF1100000
 UBOOT_ADDR_HI=0x0000007F
 ARC_MGMT_DCCM_HI=0x0000007F
 ARC_MGMT_DCCM_LO=0xFB030000
